python_modules/dagster/dagster_tests/asset_defs_tests/partition_mapping_tests/test_time_window_partition_mapping.py

```diff
@@ -397,7 +431,7 @@ def test_get_downstream_with_current_time(
             DailyPartitionsDefinition(start_date="2021-05-05", timezone="US/Central"),
             HourlyPartitionsDefinition(start_date="2021-05-05-00:00", timezone="US/Central"),
             [],
-            ["2021-05-05-23:00"],
+            ["2021-05-05-22:00"],
```

Comment:
This test case is invalid for the same reason described above.  I checked with @clairelin135 on this and she said that the parameter was actually meant to be what's here now.